Sources/Kitura/MimeTypeAcceptor.swift

```diff
@@ -62,9 +62,10 @@ extension RouterRequest {
         ///
         /// - Parameter headerValues: Array of Accept header values.
         /// - Parameter types: Array of content/mime type strings.
+        /// - Parameter wildcard: Special value that matches any value. For example "*" or "*/*"
```

Comment:
I understand what you're trying to do and it's a good fix.

I think the field/parameter needs to renamed some what to reflect that it is a "full value wildcard". Technically a mime type string of text/* is a wildcard of sorts.